PBN RCT.Rmd

```diff
@@ -507,6 +522,31 @@ pbn_complete %>%
   group_by(name) %>%
   summarize(mean = mean(value),
             sd = sd(value))
+
+#N and percent of intervention participants with sum score >3, reflect overall perceived SSI acceptability
+included_participants_pfs %>%
+  filter(treatment_group == TRUE) %>%
+  mutate(pi_pfs_sum_scores = pi_pfs_sum / 7) %>%
+  mutate(pi_pfs_sum_scores_acceptable = pi_pfs_sum_scores > 3) %>%
```

Comment:
You can create both `pi_pfs_sum_scores` and `pi_pfs_sum_scores_acceptable` in the same `mutate()`, separated by a comma